examples/supplychain-app/quorum/web3/controllers/product.js

```diff
@@ -0,0 +1,182 @@
+var Web3 = require("web3");
+var express = require("express"),
+  router = express();
+var bodyParser = require("body-parser");
+
+web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
+
+//set up the express router
+router.use(bodyParser.json()); // for parsing application/json
+const port = 8000;
+router.get("/", (req, res) => res.send("Hello World!"));
```

Comment:
this seems to be a test. If needed, add a proper message like "You have reached the correct endpoint, please use complete API paths for requests"